theano/gpuarray/opt.py

```diff
@@ -389,17 +389,18 @@ def apply(self, fgraph):
             if new_o.type != o.type:
                 assert isinstance(o.type, tensor.TensorType)
                 assert isinstance(new_o.type, GpuArrayType)
-                assert new_o.owner
```

Comment:
When we try to do something like 
```
g = GpuArrayType(dtype='float32', broadcastable=(False, False))('g')
function([g], host_from_gpu(g))
```
What happens is, the Variable in this case doesn't have an owner and this causes problem